easybuild/easyconfigs/p/Python/Python-3.7.2-fix_faulthandler.patch

```diff
@@ -0,0 +1,33 @@
+# This pull request addresses the Issue: Python 3.7.4 segfault during
+# `make test` #8771
```

Comment:
@justbennet I think it makes sense to replace the `#8771` with the full URL to the easyconfigs issue, since this patch may start "traveling" at some point... (Don't forget to change the checksum for the patch too)

Reply:
@boegel It occurred to me as I was trying to make the comment more concise, also, that perhaps the patch file should be renamed, as it is applicable to all Python 3.7.x versions both forward and backward.

So, I added explicit URLs to the patch file, renamed the patch file to be more compatible for inclusion in subsequent Python 3.7 (and 3.8 when it comes along) easyconfigs, and regenerated the sha256 checksum (thanks for the reminder to do so).

Should this also be applied to these easyconfigs?
```
Python-3.7.0-foss-2018b.eb   Python-3.7.0-iomkl-2018b.eb
Python-3.7.0-intel-2018b.eb
```
I checked manually, and the faulthandler test also fails in 3.7.0, and the same patch will prevent the failure when using GCC 8.2.0.  I tested that the patch works for Intel 18.0.5 when applied to the Python 3.7.0 source, also.

Reply:
@justbennet It makes sense to also apply this patch for Python 3.7.0, but should we do that in a separate PR (so we can merge this one, and include it in the EasyBuild v3.9.4 release which will go out really soon)?

Reply:
I think getting this one done for the next release and then adding a second PR for the later release might be the better course.

Reply:
:+1: